packages/mdjs-core/src/mdjsProcess.js

```diff
@@ -54,6 +55,11 @@ const defaultMetaPlugins = [
  */
 async function mdjsProcess(mdjs, { setupUnifiedPlugins = [] } = {}) {
   const parser = unified();
+  if (!rehypePrism) {
+    rehypePrism = (await import('rehype-prism')).default;
+    loadLanguages(['md', 'sh', 'yml']);
```

Comment:
These look to be missing from the defaults of this package, if there are other languages used that you think might be left out, point me in that direction.